src/main/java/at/pfeifer/chatapp/controller/ChatController.java

```diff
@@ -30,14 +30,21 @@ public class ChatController implements Initializable {
     @FXML
     private Button sendButton;
 
+    @FXML
+    private TextField userName;
+
+    String userNameText;
```

Comment:
```suggestion
@FMXL
private TextField signatureInput
```
Change to signatureInput
And remove "userNameText" variable because its unneeded